article/sections/_0_abstract.tex

```diff
@@ -1 +1,9 @@
 \section*{Abstract}
+
+In this chapter, we focus on using causal graphs to make causal inferences in choice modelling contexts.
+We address a longstanding disconnect whereby choice modellers have not adopted techniques from causal inference researchers, even when trying to make causal inferences.
+To bridge this disconnect, we conduct simulation studies to first demonstrate the need for paying close attention to causal graphs in choice modelling.
+We then present new guidelines, methods, and perspectives for the construction and validation of causal graphs, complete with empirical examples.
+Next we give examples and direction for using one's causal graphs to make causal inferences in common choice modelling scenarios, including those with latent confounding.
+Finally, throughout the chapter, we point readers to more information through extensive literature reviews.
```

Comment:
I would add examples of what information we're pointing them to.

Reply:
Do you have a specific suggestion in mind? I ask because I'm having a hard time thinking of something that is succinct and also not overly limiting / restrictive.

Reply:
Never mind I got it! Thanks for the call out!

Reply:
Yea the change looks great!